src/github.com/getlantern/lantern-mobile/app/src/main/res/values/styles.xml

```diff
@@ -15,11 +15,6 @@
         <item name="color">#39C2D6</item>
     </style>
 
-    <style name="Theme.HomeScreen" parent="android:Theme.Holo.NoActionBar.TranslucentDecor">
```

Comment:
This wasn't necessary for the fix, but appeared to be dead code so I removed it.